docs/lang/articles/contribution/windows_debug.md

```diff
@@ -0,0 +1,93 @@
+## Prerequisites:
+Able to build Taichi from source. (i.e. already have LLVM and related environment variables already setup)
+Recommended plugins to install for Visual Studio:
+1. Github Co-pilot
+2. VS Chromium for Code Search ( https://chromium.github.io/vs-chromium/ ). You might need to go to the release page to find the pre-release version needed for VS2022
+
+## Step 1. Turn on msbuild option in our build system:
+
+This is a new feature in setup.py (introduced in https://github.com/taichi-dev/taichi/pull/6724 ) that enables building Taichi with MSBuild & MSVC, and it generates a Visual Studio Project file to enable coding and seamless debugging through Visual Studio IDE.
+To turn it on, create a new environment variable (Type in windows search: environment variable, or set it temporarily with `$Env:` in PowerShell) called TAICHI_USE_MSBUILD, and set its value to 1 or ON.
+Now after cleaning up the _skbuild folder (in case the previously used generator is Ninja), run `python setup.py develop` and build Taichi from the source.
+
+## Step 2.  Navigate to the generated Visual Studio Project file
+Go to the build folder, in my case it is `_skbuild\win-amd64-3.9\cmake-build`, and double click on `taichi.sln` (Or open this SLN solution file from Visual Studio). This should open Visual Studio IDE with the setup for Taichi.
+After Visual Studio launched, you should see something like this (layout might be different, you can reconfigure the layout to one that you like):
+
+![image13](https://user-images.githubusercontent.com/11663476/212577220-92a8a7cb-f6ff-4365-9808-0a7299be87cd.png)
+
+## Step 3. Navigating the source code of Taichi in Visual Studio
+
+The top right red box is the solution explorer, where you can see each submodule of Taichi grouped up by each “object file”, clicking on one will reveal something like this:
+
+![image10](https://user-images.githubusercontent.com/11663476/212577350-33912d21-0105-459b-8490-2aaee5c88ff6.png)
+![image11](https://user-images.githubusercontent.com/11663476/212577355-fec6837a-00fc-4f7b-8cdc-b369bc4bc015.png)
+
+Or if you prefer navigating the project using its folder structure, you can find the “Source Explorer” module, and it should reveal the normal Taichi source directory structure:
```

Comment:
Are the quotes Chinese characters?

Reply:
No, they are not. English also has left and right quotation marks.